README.md

```diff
@@ -372,9 +382,13 @@ you should use any version `<5.5 >=5.7.1` as the versions in-between have some n
 
 The NPM packager supports the following `packagerOptions`:
 
-| Option             | Type | Default | Description                                         |
-| ------------------ | ---- | ------- | --------------------------------------------------- |
-| noInstall          | bool | false   | Do not run `npm install` (assume install completed) |
+| Option             | Type   | Default               | Description                                         |
+| ------------------ | ------ | --------------------- | --------------------------------------------------- |
+| noInstall          | bool   | false                 | Do not run `npm install` (assume install completed) |
+| lockFile           | string | ./package-lock.json   | Relative path to lock file to use                   |
+
+When using NPM version `>= 7.0.0`, we will use the package lock file instead of the installed node_modules. This improves the 
```

Comment:
```suggestion
When using NPM version `>= 7.0.0`, we will use the `package-lock.json` file instead of modules installed in `node_modules`. This improves the 
```